frontend/src/pages/ChainMemberList.tsx

```diff
@@ -249,15 +250,29 @@ function HostTable(props: {
   }
 
   function onDemote() {
-    if (!selected) return;
-    let chainUID = props.chain.uid;
```

Comment:
Place all required values as variables here, to ensure that nothing could change between *opening the dialogue* and *clicking executing the action*